verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java

```diff
@@ -140,6 +144,47 @@ public void notifyCommiter(
         if(subject.equals("Verigreen Status - Failure"))
         {
         	status.append(String.format("<tr><td>Temporary branch:</td><td>%s</td></tr></table>", tempBranch));
+        	if(offlineMode.equals("false"))
+        	{
```

Comment:
@borzamircea  - please put this entire block in a method that receive the verification status as a param.